app/services/foreman/renderer/scope/macros/snippet_rendering.rb

```diff
@@ -10,9 +10,20 @@ module SnippetRendering
             sections only: %w[all provisioning]
           end
 
+          apipie :method, 'Checks if a snippet exitsts in template source, e.g. database' do
+            desc 'Checks if a snippet exists, does not render template.'
+            required :name, String, desc: 'Name of the snippet template to check for'
+            returns [true, false], desc: 'presence of the snippet'
+            example "snippet_exist('motd') # => true"
+            see 'snippet', description: 'Snippet#snippet', scope: Foreman::Renderer::Scope::Macros::SnippetRendering
+          end
+          def snippet_exists(name)
```

Comment:
Methods that return a boolean should end with `?`
```suggestion
          def snippet_exists?(name)
```